app/helpers/distribution_helper.rb

```diff
@@ -32,4 +33,8 @@ def quantity_by_item_category_id(distribution, item_category_id)
 
     quantities[item_category_id]
   end
+
+  def distribution_shipping_cost(shipping_cost)
+    shipping_cost.present? ? number_to_currency(shipping_cost) : "0.00"
```

Comment:
Can we leave it blank if there is no shipping cost rather than showing 0.00?

Reply:
That sounds better to me, too -- so far we only have a couple banks who have mentioned shipping.

Reply:
done